pkg/operator/ceph/object/spec.go

```diff
@@ -104,6 +105,36 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) v1.PodTemplateSpec
 	return podTemplateSpec
 }
 
+func (c *clusterConfig) setPodPlacement(pod *v1.PodSpec, p rook.Placement) {
+	p.ApplyToPodSpec(pod)
+
+	// label selector for gateways used in anti-affinity rules
+	monAntiAffinity := v1.PodAffinityTerm{
+		LabelSelector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				k8sutil.AppAttr: AppName,
+				"rgw":           c.store.Name,
+			},
+		},
+		TopologyKey: v1.LabelHostname,
+	}
+
+	// set gateways pod anti-affinity rules. when gateways should never be
+	// co-located (e.g. HostNetworking) then the
+	// anti-affinity rule is made to be required during scheduling
+	//
```

Comment:
This comment block can probably be reduced to 3 lines total if we wrap at 100 lines. Also, remove blank comment line, and add period to end of line above.